cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -184,6 +184,8 @@ spec:
             - HELMSUBST_DEPLOYMENT_AUDIT_LOGFILE
             - --disable-cert-rotation={{ or .Values.audit.disableCertRotation .Values.externalCertInjection.enabled }}
             - --external-data-provider-response-cache-ttl={{ .Values.externaldataProviderResponseCacheTTL }}
+            - --experimental-enable-k8s-native-validation={{ .Values.enableK8sNativeValidation }}
+            - --vap-enforcement={{ .Values.vapEnforcement }}
```

Comment:
@ritazh will the change contemplated by

https://github.com/open-policy-agent/frameworks/pull/427

affect this flag?

I don't think it will b/c we are assuming undefined on the template will default to the behavior provided by the flag?

Reply:
Right. undefined on the template will default to this flag. scopedEnforcementAction at the constraint level can however omit the constraint from audit is a new experience that may be surprising to users.